astero/private/adipls_support.f90

```diff
@@ -683,7 +683,7 @@ subroutine read_and_store(iriche, iturpr, cgrav)
             call mesa_error(__FILE__,__LINE__)
          end if
          call fgong_amdl( &
-            cgrav, nn, iconst, ivar, ivers, glob, var, data, aa, nn, ierr)
```

Comment:
If you look above this line, `nn` is uninitialized and this should be `nn_in` since this function argument has `intent(in)`

Reply:
Agreed. `nn_in` is populated by `read_fgong_file`, even though the variable name is misleading.